web-components/src/api/robotoff.ts

```diff
@@ -0,0 +1,48 @@
+import { ROBOTOFF_API_URL, IS_DEVELOPMENT_MODE } from "../constants"
+import { addParamsToUrl } from "../utils"
+import { getLocale } from "../localization"
+import {
+  QuestionRequestParams,
+  QuestionsResponse,
+  QuestionAnnotationAnswer,
+} from "../types/robotoff"
+
+const robotoff = {
+  annotate(insightId: string, annotation: QuestionAnnotationAnswer) {
+    const formBody = new URLSearchParams({
+      insight_id: insightId,
+      annotation: annotation,
+    }).toString()
+    if (IS_DEVELOPMENT_MODE) {
```

Comment:
Instead of IS_DEVELOPMENT_MODE, I propose to rely on a specific attribute (dry-run ?) or something better.
It's because we will want to test integration while developing (it can even be on a local robotoff instance)